zfsbootmenu/bin/zsnapshots

```diff
@@ -19,18 +20,61 @@ done
 
 unset src sources
 
+list_root_filesystems() {
```

Comment:
Is this worth trying to merge with the largely similiar process in `populate_be_list()` in `zfsbootmenu-ui.sh`, so that if the logic for determining a root filesystem changes in the future it doesn't have to be updated in multiple places?

Reply:
Good point. I moved the "BE candidate" logic into a common function.